applications/StabilizedCFDApplication/custom_elements/dss_fic.cpp

```diff
@@ -675,8 +677,9 @@ double DSS_FIC<TDim>::ProjectedSizeHexa(const array_1d<double,3> &rDirection)
     }
 
     Matrix QInv = ZeroMatrix(3,3);
```

Comment:
I think that you don't need to set Qinv to zero, it will be overwritten internally